### DISCUSSION TITLE
Is it possible to use Devuan as a base?? ((Pure Debian upstream but no System-D))

### DISCUSSION THREAD

**Discussion by victoroldschool** [SpiralLinux/SpiralLinux-project#429] [General]
Hey, I love this project but couldn't find a way to contact you. I see your OS is using pure Debian upstream, and I'm wondering if it would be easy to use your setup/tweaks with a Devuan base?? ((it's also pure Debian upstream but comes without system-d -- instead it comes with either sysvinit, runit, openrc)). I'm hoping you might have a script that you use to apply all the patches that I could utilize.

Ideally, I'd like to be able to either integrate or apply your patches/mods to either a base server/desktop image or install. 

I noticed one of the recent changes in the changelog involved system-d so figured it best to ask.  I prefer running a pure ALSA system for the audio, no extra pulseaudio layer... So far I've been able to achieve this via a Devuan server install + LxQT.

Thanks very much!

**Comment by geckolinux** [comment]
Hi there, thanks for the question. In theory it should be possible to build an installable live system from Devuan sources. I use Debian's [live-build](https://live-team.pages.debian.net/live-manual/html/live-manual/index.en.html) tool to build the ISOs, and I upload the full build directory structure for each release here:
https://github.com/SpiralLinux/SpiralLinux-project

It looks like you could use the same tool to build a Devuan image by changing the source repositories, according to these results from a quick search:
- https://dquinton.github.io/debian-install/netinstall/live-build.html
- https://forums.bunsenlabs.org/viewtopic.php?id=710

Here's a bit more information about how you can build images based on SpiralLinux:
- https://github.com/orgs/SpiralLinux/discussions/321#discussioncomment-7235501
- https://github.com/orgs/SpiralLinux/discussions/228#discussioncomment-4809266

**Reply by victoroldschool** [reply]
Thanks so much for the kind response, I really appreciate it! And I’m gonna do just that and start reading up on the resources you posted. Yess that’s EXACTLY what I was hoping I could do ((change the source repos)).

This should be a viable option, thanks very much!

**Reply by crhy** [reply]
Any prorgress Victor?  I'm thinking of a very similar thing....
